include/pybind11/detail/internals.h

```diff
@@ -311,12 +311,23 @@ PYBIND11_NOINLINE inline internals &get_internals() {
     return **internals_pp;
 }
 
+
+struct module_internals {
```

Comment:
Can/should we make this `module_local_internals`?

Reply:
(same for `get_module_local_internals`, then, if we do)

Reply:
I'd do either `local_internals` or `module_internals` to contrast with the normal `internals`. I don't see a reason to give it all the names. but if that is what people want, I'll do it

Reply:
I was thinking of "module local" being a concept, but it's true it also wasn't done for `registered_local_types_cpp`. Maybe matching `local` is more important than `module`, though?

Let's see what others say.

Reply:
currently went with `local_internals` and `get_local_internals`. obviously easy to change depending on what everyone wants